content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -191,10 +191,6 @@ resource limits, see the
 
 The resource usage of a Pod is reported as part of the Pod status.
 
-If [optional monitoring](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/cluster-monitoring/README.md)
-is configured for your cluster, then Pod resource usage can be retrieved from
-the monitoring system.
-
```

Comment:
Maybe signpost readers to https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/?

Reply:
That is actually a very good idea. I rephrased the original sentence and added to links to the page and the metrics api.